modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -2994,6 +2997,44 @@ public void testMasterKeyChangeOnInactiveCluster() throws Exception {
         assertContains(log, testOut.toString(), "Master key change was rejected. The cluster is inactive.");
     }
 
+    /** @throws Exception If failed. */
+    @Test
+    @WithSystemProperty(key = SNAPSHOT_LIMITED_TRANSFER_BLOCK_SIZE, value = "1")
```

Comment:
Since `SNAPSHOT_TRANSFER_BLOCK_SIZE_BYTES` is static, this `WithSystemProperty` does nothing (unless this test is the first after JVM start).

Reply:
System property removed.